package.json

```diff
@@ -199,6 +201,8 @@
     "@ionic-native-mocks/qr-scanner": "2.0.7",
     "@ionic-native-mocks/touch-id": "2.0.7",
     "@ionic/app-scripts": "3.1.8",
+    "@ionic/cli-plugin-cordova": "1.6.2",
+    "@ionic/cli-plugin-ionic-angular": "1.4.1",
```

Comment:
I got a warning that these packages aren't necessary if I have a updated cli. I think that it 's safe to remove them.

```
[WARN] Detected @ionic/cli-plugin-cordova in your package.json.

       As of CLI 3.8, it is no longer needed. You can uninstall it:
       
       npm uninstall -D @ionic/cli-plugin-cordova
       

[WARN] Detected @ionic/cli-plugin-ionic-angular in your package.json.

       As of CLI 3.8, it is no longer needed. You can uninstall it:
       
       npm uninstall -D @ionic/cli-plugin-ionic-angular
```